src/content-cli-analyze.ts

```diff
@@ -0,0 +1,43 @@
+import { ContextInitializer } from "./util/context-initializer";
+import { logger } from "./util/logger";
+import { ActionFlowCommand } from "./commands/action-flow.command";
+import commander = require("commander");
+
+type CommanderStatic = commander.CommanderStatic;
+
+class Analyze {
+    public static actionFlows(program: CommanderStatic): CommanderStatic {
+        program
+            .command("action-flows")
+            .description("Analyze Action Flows dependencies for a certain package")
+            .option("-p, --profile <profile>", "Profile which you want to use to analyze Action Flows")
+            .requiredOption("-i, --packageId <packageId>", "ID of the package from which you want to export Action Flows")
+            .option("-o, --outputToJsonFile", "Output the analyze result in a JSON file")
```

Comment:
Let's name this option `--json` to stay consistent with other commands.
```suggestion
            .option("--json", "Return response as json type", "")
```

Edit: Please apply the comments for all the commands.

Reply:
I tried to follow the other commands in `import` and `export`
- export data-pool - https://github.com/celonis/content-cli/blob/141ad700180f8f2a89dad80ac35b0baac312dc90/src/content-cli-export.ts#L31
- import data-pools - https://github.com/celonis/content-cli/blob/141ad700180f8f2a89dad80ac35b0baac312dc90/src/content-cli-import.ts#L36

I can see that `--json` is used at `config` and `list` commands, but I'm not sure if we should be consistent with that, or with the other commands in the same file

Reply:
hmmm, I guess we already have this format for this command. Let's leave like this for now, we can look into unifying the options in the future.